cmdstanpy/model.py

```diff
@@ -122,8 +122,8 @@ def __init__(
                 libtbb = os.path.join(
                     cmdstan_path(), 'stan', 'lib', 'stan_math', 'lib', 'tbb'
                 )
-            os.environ["PATH"] = ";".join(list(OrderedDict.fromkeys(
-                [libtbb, ] + os.getenv("PATH", "").split(";")
+            os.environ['PATH'] = ';'.join(list(OrderedDict.fromkeys(
+                [libtbb, ] + os.getenv('PATH', '').split(';')
```

Comment:
Splitting by `;` is correct thing here. I think there are cases when the path/filename might contain ; but it is not our "problem"

Reply:
agreed - flake8 complained about double quotes, so changed to single.